contracts/BaseStrategy.sol

```diff
@@ -139,6 +139,18 @@ abstract contract BaseStrategy {
     // So indexers can keep track of this
     event Harvested(uint256 profit, uint256 loss, uint256 debtPayment, uint256 debtOutstanding);
 
+    event StrategistUpdated(address newStrategist);
+
+    event KeeperUpdated(address newKeeper);
+
+    event RewardsUpdated(address rewards);
+
+    event ReportDelayUpdated(uint256 delay);
+
+    event ProfitFactorUpdated(uint256 profitFactor);
+
+    event DebtThresholdUpdated(uint256 debtThreshold);
+
```

Comment:
Can reverse the wording here (`Update*`) so it is easier to grok?

Reply:
done on latest commits